lib/src/bindings/bindings.dart

```diff
@@ -284,6 +308,31 @@ class _ObjectBoxBindings {
 
     obx_query_visit = _fn<obx_query_visit_native_t>("obx_query_visit").asFunction();
 
+    // query property
+    obx_query_prop = _fn<obx_query_prop_t<Uint32>>("obx_query_prop").asFunction();
+    obx_query_prop_close = _fn<obx_query_prop_close_t<Int32>>("obx_query_prop_close").asFunction();
+
+    obx_query_prop_distinct =        _fn<obx_query_prop_distinct_t<Int32, Int8>>("obx_query_prop_distinct").asFunction();
+    obx_query_prop_distinct_string = _fn<obx_query_prop_distinct2_t<Int32, Int8>>("obx_query_prop_distinct_string").asFunction();
+
+    obx_query_prop_count = _fn<obx_query_prop_op_t<Int32, Uint64>>("obx_query_prop_string_find").asFunction();
+    obx_query_prop_avg = _fn<obx_query_prop_op_t<Int32, Double>>("obx_query_prop_avg").asFunction();
+    obx_query_prop_min = _fn<obx_query_prop_op_t<Int32, Double>>("obx_query_prop_min").asFunction();
+    obx_query_prop_max = _fn<obx_query_prop_op_t<Int32, Double>>("obx_query_prop_max").asFunction();
+    obx_query_prop_sum = _fn<obx_query_prop_op_t<Int32, Double>>("obx_query_prop_sum").asFunction();
+
+    obx_query_prop_min_int = _fn<obx_query_prop_op_t<Int32, Int64>>("obx_query_prop_min_int").asFunction();
+    obx_query_prop_max_int = _fn<obx_query_prop_op_t<Int32, Int64>>("obx_query_prop_max_int").asFunction();
+    obx_query_prop_sum_int = _fn<obx_query_prop_op_t<Int32, Int64>>("obx_query_prop_sum_int").asFunction();
+
+    obx_query_prop_string_find = _fn<obx_query_prop_find_t<OBX_string_array, Int8>>("obx_query_prop_string_find").asFunction();
```

Comment:
to get rid of code analysis reports (the same for the remaining finds())
```suggestion
    obx_query_prop_string_find = _fn<obx_query_prop_find_native_t<Pointer<OBX_string_array>, Int8>>("obx_query_prop_string_find").asFunction();
```